app/jobs/process_pull_request_update_job.rb

```diff
@@ -0,0 +1,7 @@
+class ProcessPullRequestUpdateJob < ApplicationJob
+  queue_as :default
+
+  def perform(action, author, url, html_url)
```

Comment:
```suggestion
  def perform(action, author, pr_url, html_url)
```

(I'm guessing that's what this is?)

Reply:
It is the PR's API URL. So maybe either:

`def perform(action, author, api_url, html_url)`

or else

`def perform(action, author, pr_url, pr_html_url)`

?

Reply:
Yeah, `api_url` then I think.